bin/odyssey/src/main.rs

```diff
@@ -121,3 +133,26 @@ fn main() {
         std::process::exit(1);
     }
 }
+
+fn init_sponsor() -> eyre::Result<(Option<EthereumWallet>, Option<Address>)> {
```

Comment:
nit, can we introduce a type for this, so we only need to return `Option<Sponsor>`?